hack/ci/e2e-k8s.sh

```diff
@@ -62,6 +62,15 @@ build() {
   make all WHAT='cmd/kubectl test/e2e/e2e.test vendor/github.com/onsi/ginkgo/ginkgo'
 }
 
+check_log_format() {
```

Comment:
nit: this method name doesn't seem very clear, it's only checking the Kubernetes version and erroring and only used for KUBELET_LOG_FORMAT

Reply:
I have renamed it as check_structured_log_support to better reflect the operation.